book/src/faq.md

```diff
@@ -110,6 +110,9 @@ method.
 For more information about all the possible values for `emits_changed_signal` refer
  to [`dbus_proxy`](https://docs.rs/zbus/3.13.1/zbus/attr.dbus_proxy.html) documentation.
 
+[`ProxyBuilder::uncached_properties`]: https://docs.rs/zbus/3/zbus/struct.ProxyBuilder.html#method.uncached_properties
+[`ProxyBuilder::cache_properties`]: https://docs.rs/zbus/3/zbus/struct.ProxyBuilder.html#method.cache_properties
```

Comment:
It doesn't make sense to add (unused) links in one commit and then only use them in the following commit. They're the same logical commit.

Reply:
I've added all the URL reference changes into 1 commit